docs/accessibility/accessibility-module.md

```diff
@@ -47,7 +136,9 @@ This behavior can be configured with the [`accessibility.linkedDescription`](htt
 ```
 <iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/highcharts/accessibility/accessible-avg-temp allow="fullscreen"></iframe>
 
-**Caption:**
+[View demo code](https://jsfiddle.net/gh/get/library/pure/highcharts/tree/master/samples/highcharts/accessibility/accessible-avg-tmp)
```

Comment:
Does not work

Reply:
Do you mean the demo is not working?

Reply:
I get a 404:
```
404

Repository/Gist not found: https://api.github.com/repos/highcharts/contents/samples/highcharts/accessibility/accessible-avg-tmp
```

Reply:
I see! URL had a typo, should be fixed now